### ISSUE TITLE
Find widget focus movement could be improved

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#116450]
I am looking into improving the tab focus navigation across the workbench such that each ActionBar is one tab, and inside it you would have to user arrow navigation to move focus.
I have run across the find widget and there I see we do quite some custom solutions. So wanted to first create this issue to show what currently does not work (independent of changes which I would like to do).

1. Open find widget
2. You can not focus onto `Find In Selection` button 🐛 
3. You can use `left` / `right` arrow navigation inside the find input, but not for the find widget buttons on the right 🐛

**Comment by isidorn** [comment]
Ideally we would change the `findWidget` to use the `ActionBar`. Since most of these are checkboxes the `keybindingsEditor` already does a good example of this using `checkboxViewItems` [here](https://github.com/microsoft/vscode/blob/6ee4ef1c2dbbe5a8763ccdce796a40be84c7ee8a/src/vs/workbench/contrib/preferences/browser/keybindingsEditor.ts#L395)

**Comment by isidorn** [comment]
@rebornix also let me know if you need help here
Nice things is that the findInput is shared across findWidget and search so we should just fix that in one place. The only remaining things to make the find widget nice are the actions outside of the input

fyi @JacksonKearl

**Comment by roblourens** [comment]
This seems weird, I can shift+tab to "Find in Selection", but not forward tab. And I can use arrow keys to move focus in the toolbar, but those buttons are also all tab stops.

**Comment by starball5** [comment]
Somewhat related: #193331
